featuretools/entityset/deserialize.py

```diff
@@ -161,7 +163,6 @@ def read_entity_data(description, path):
     else:
         error = 'must be one of the following formats: {}'
         raise ValueError(error.format(', '.join(FORMATS)))
-    dtypes = description['loading_info']['properties']['dtypes']
```

Comment:
Changing the way the Dask entityset was created was causing a Dask error in `test_to_csv`. Dask was inferring the `log` entity `zipcode` column as `int64`, but it should be an `object` column based on the data it contains. The changes here were needed to tell Dask what the column types should be when reading the CSV, so it doesn't try to infer them. This is probably an overall safer way to read the file anyway since we know from serialization what the column dtypes should be.

I'm not exactly sure why changing the entityset construction highlighted this, but it did...

Also, I had to change the way the `dtypes` variable was created above because one of the tests for invalid input does not contain the `properties` entry in `loading_info` so using the old selection method resulted in a `KeyError`.